config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  root to: "homepages#index"
+
+  resources :drivers 
+  resources :passengers do
+    resources :trips, only: :create
+  end
+  resources :trips, expect: [:index, :new]
```

Comment:
Good work limiting your routes. Small error, this should be:
```suggestion
  resources :trips, except: [:index, :new]
```

Reply:
Aha-ha! Good catch!